pootle/apps/pootle_statistics/models.py

```diff
@@ -325,10 +325,7 @@ def save(self, *args, **kwargs):
 
         scorelogs_created = []
         for score in ScoreLog.get_scorelogs(submission=self):
-            if 'action_code' in score and score['user'] is not None:
-                if not isinstance(score["user"], models.Model):
-                    score["user_id"] = score["user"]
-                    del score["user"]
+            if 'action_code' in score and score['user_id'] is not None:
```

Comment:
@ta2-1 in my testing sometimes the user was a real user object - in these cases it saves sql to pass it through